installer/modules/commons.py

```diff
@@ -78,4 +88,10 @@ def replace_string_in_file(filename,  search_string,  replace_string):
 
     with open(filename, "w") as destination:
         for line in lines:
-            destination.write(re.sub(search_string,  replace_string,  line))
\ No newline at end of file
+            destination.write(re.sub(search_string,  replace_string,  line))
+
+def log(type, message):
+    command = 'systemd-cat echo \"<{}>{}\"'.format(type, message)
```

Comment:
Numbers is hard to understand in the logs, can you add a string describe the error level.